src/components/Features/Features.css

```diff
@@ -0,0 +1,92 @@
+.features {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 62px 50px;
+  gap: 10px;
+}
+
+.featureHead {
+  color: #ffffff;
+}
+
+.head {
+  text-align: center;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 52px;
+  line-height: 62px;
+  letter-spacing: 0.2px;
+}
+
+.featuresHeader {
+  width: 100%;
+  padding: 10px;
+}
+
+.featureSubHead .featureText {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 30px;
+  text-align: center;
+  letter-spacing: 0.2px;
+}
+
+.featureSubHead {
+  text-align: center;
+  padding: 15px;
+}
+.featuresFrame {
+  display: flex;
+  padding: 50px 130px;
+}
+.frameContent {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 40px 20px;
+}
+.contentRow {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 25px;
+}
+
+.rowTitle {
+  display: flex;
+  align-items: center;
+}
+
+.title {
+  color: #ffffff;
+  padding: 10px;
+  width: 65%;
+}
+
+.infinity .title {
+  width: 75%;
+}
+
+.frame {
+  min-width: 65%;
+}
+
+.frameImg {
+  width: 100%;
+}
+
+.rowText {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  letter-spacing: 0.2px;
+  color: #ffffff;
+  width: 75%;
```

Comment:
There is inconsistency in the uses of `<span>` and `<p>`
stick with one.

also considering that a card is a component. you could
use group styling. instead of styling every individual title for every
card. 

if another card were to be added it should automatically assume the proper styling.